src/main/java/com/epam/reportportal/junit5/ReportPortalExtension.java

```diff
@@ -192,10 +190,14 @@ public void beforeTestExecution(ExtensionContext context) throws Exception {
         if (isTemplateTestInvocation(context)) {
             // if not yet done, start containing suite
             Maybe<String> suiteId = startSuiteIfRequiredFor(context);
+            // set test item start time
+            rq.setStartTime(Calendar.getInstance().getTime());
             // start test item for this invocation
             itemId = launch.startTestItem(suiteId, rq);
         // otherwise (not template test invocatoin)
         } else {
+            // set test item start time
+            rq.setStartTime(Calendar.getInstance().getTime());

```

Comment:
no need to put it into each if/else block, it could be placed before if

Reply:
This is actually necessary because we may need to add a suite container for repeated tests. The back-end rejects any test that has a start time that precedes the start time of the suite that contains it.